blivet/formats/luks.py

```diff
@@ -105,6 +105,9 @@ def __init__(self, **kwargs):
                 # default to the max (512 bits) for aes-xts
                 self.key_size = 512
 
+        if not self.exists and self.key_size is None:
+            self.key_size = 0
+
```

Comment:
Maybe we should instead change the `kwargs.get("key_size)` to `kwargs.get("key_size") or 0` if the default of None isn't useful.

Reply:
You can do ``kwargs.get("key_size", 0)``

Reply:
Fixed.

Reply:
Using `dict.get` alone will work as long as nobody passes `key_size=None`.

Reply:
Fixed.

Reply:
> Using dict.get alone will work as long as nobody passes key_size=None.

Good point!